demos/text-field.html

```diff
@@ -358,19 +361,26 @@ <h2>CSS Only Text Field</h2>
           <div class="mdc-text-field" data-demo-no-auto-js>
             <input type="text" class="mdc-text-field__input" id="css-only-text-field"
                    placeholder="Name">
-            <div class="mdc-text-field__bottom-line"></div>
           </div>
           <label for="css-only-text-field">Your name:</label>
         </div>
 
         <div>
-          <input id="css-only-tf-disabled" type="checkbox">
+          <input id="css-only-tf-disabled" class="demo-disabled" type="checkbox">
```

Comment:
Curious why `demo-` prefix is used on the classes for disabled and dense checkboxes when it's not used for the others?